test/integration/resources/cps_certificate/create.tf.step

```diff
@@ -0,0 +1,101 @@
+resource "edgecast_cps_certificate" "certificate_1" {
+	certificate_label = "Terraform Integration Test 013"
```

Comment:
The label needs to be passed in as a var, maybe generated from a timestamp in YYYYMMDDhhmmss format (I believe colons (:) are not allowed). Otherwise new tests will fail because the label is not unique.

Reply:
Or perhaps the `timestamp` function in TF: https://www.terraform.io/language/functions/timestamp